test/end-to-end/resetpasswordusingtoken.test.js

```diff
@@ -118,6 +121,25 @@ describe("SuperTokens Reset password", function () {
             await page.goto(`${TEST_CLIENT_BASE_URL}/auth/reset-password`);
         });
 
+        it("Should redirect to Sign In screen when back button is clicked", async function () {
+            const backButton = await getResetPasswordFormBackButton(page);
+
+            // assert that the back button exists
```

Comment:
No need for these, the `waitForSTElement` already checks and always returns a value if you didn't set it as inverted.

Reply:
Updated